json/src/moditect/module-info.java

```diff
@@ -13,12 +13,12 @@
 
     requires com.fasterxml.jackson.jaxrs.base;
 
-    requires static javax.ws.rs.api;
+    requires static jakarta.ws.rs.api;
```

Comment:
I think this line and all similar in other `module-info.java` files should not change. See #127 and #121